DATABASE/stored_procedures/orders/createNewOrder.sql

```diff
@@ -4,8 +4,8 @@ CREATE OR ALTER PROCEDURE createOrder(
 
 AS 
 BEGIN 
-INSERT INTO orders (id,userID,orderDate)
-VALUES (@id, @userID,@orderDate)
+INSERT INTO orders (id,userID)
+VALUES (@id, @userID)
 UPDATE orders SET  orderStatus='processing'
```

Comment:
INSERT INTO orders (id, userID, orderStatus)
VALUES (@id, @userID, 'processing')

This update will change all orders to processing.

Reply:
what should I do ? remove it

Reply:
yeah remove update
then no need to insert orderStatus because it will have default